docs/components.rst

```diff
@@ -1,2 +1,63 @@
 Components
-==========
\ No newline at end of file
+==========
+
+Everything which is required to provision a virtual machine or docker container are wrapped under a term called components.
+Read further for complete details.
+
+Infrastructure
+--------------
+This includes information regarding hardware availability of remote machine.
+
+++++++++++++++++
+Compute Resource
+++++++++++++++++
+
+Compute Resource is very first step in provisioning virtual machines and runnig Docker containers.
+
++ Create New
+
+    Firstly you have to create your own compute resource it will ask three fields
```

Comment:
remove **it will ask three fields**

Reply:
Should we use this line instead of the above one
'Initially we need to add compute which includes following parameters'

Reply:
yes